### ISSUE TITLE
Run Nginx outside of Docker for file service.

### ISSUE THREAD

**Issue by shirey** [hubmapconsortium/ingest-ui#116]
Won't do, not needed

**Comment by yuanzhou** [comment]
Would it be an alternative if we include  the TMS's name in the file URL? For example, currently we use this URL pattern `https://assets.dev.hubmapconsortium.org/<dataset-uuid>/path-to-file.ext`, it'll look like `https://assets.dev.hubmapconsortium.org/<TMC-name>/<dataset-uuid>/path-to-file.ext`. And doing this allows us to service the file directly by nginx container after file auth service check, without using symlink at all.

Two reasons I'm thinking about this approach:
1. running nginx outside docker will certainly work with the symlink, it adds more changes to the current github code structure. And no changes needed if we add the `<TMC-name>`.

2. it's possible we may need to move the original directories or files in the future, we'll have to also update those symlinks manually, and it can introduce new bugs.

Basically, the `<TMC-name>` in the file URL won't be used by gateway's file auth check, we are still only using the globus token and the `<dataset-uuid>` as well as the gene sequences attribute for permission check. And the Harvard UI team shouldn't care about how the file URL looks. We generate each dataset/file URL for them.

@shirey any insights?
